skins/zwall.scss

```diff
@@ -0,0 +1,23 @@
+.zwap {
```

Comment:
Should `zwall` selector

Reply:
https://github.com/Zilliqa/snapshot-spaces/pull/30/commits/7d5afe4aef105a072fd4b65dea606305a043b26d